packages/server-ct/test/unit/project-ct.spec.ts

```diff
@@ -11,3 +11,35 @@ describe('Project', () => {
     expect(typeof p.close).to.equal('function')
   })
 })
+
+describe('addComponentTestingUniqueDefaults', () => {
+  it('overrides e2e defaults if no user overrides provided', () => {
+    const expected = {
+      viewportHeight: 500,
+      viewportWidth: 500,
+    }
+    const p = new ProjectCt('foo')
+
+    const actual = p.addComponentTestingUniqueDefaults({
+      viewportHeight: 660,
+      viewportWidth: 1000,
+    })
+
+    expect(actual).to.eql(expected)
```

Comment:
Not a blocker, just an idea -- when we have a small expected it may have sense to inline it for readability. So it's not needed to scroll up or move eyes back to understand the assertion. What do you think?

Reply:
Sure